### DISCUSSION TITLE
4 requests per second and more storage per basket?

### DISCUSSION THREAD

**Discussion by WallOfMessages** [imRohan/Pantry#174] [Q&A] [answered]
100 mb of space is cool and all but it's distributed evenly among 100 bins. maybe a setting that allows us to change how much storage a bin gets? also 4 request per second might be a stretch but 2 request per second is just hella slow.

**Comment by imRohan** [comment, marked as answer]
Rate limiting aside everything seems totally doable! I like your idea for adjusting the allotment sizes, it would be interesting to build that out.

2 Requests a second is (at this point) a bit of a necessity due to limitations with the server and its size. Pantry is a free service for users that I pay for out of pocket and I'm just not ready to bump up the server size without implementing a paid tier.
